README.md

```diff
@@ -1 +1,5 @@
-# project-rl
\ No newline at end of file
+# Project-reinforcement-learning
+
+Tracking reinforment learning experiments using Neptune.
+
+[Explore the project in the Neptune app](https://app.neptune.ai/o/common/org/project-rl/experiments?split=tbl&dash=parallel-coordinates-plot&viewId=167514ce-1d4f-4954-a9c0-baea4e2ca42c).
```

Comment:
Just curious as to why/how the link mentions `parallel-coordinates-plot` but correctly opens on runs table.

Reply:
thats because `dash=parallel-coordinates-plot` is only valid when `split=cmp`. Since the runs table does not have a parallel coordinates plot, it defaults to the table